backend/migrations/20241216163427_first-kemomimi.down.sql

```diff
@@ -0,0 +1,11 @@
+-- 製品-カテゴリ関係テーブル（ProductCategory）削除
+DROP TABLE ProductCategory;
```

Comment:
テーブルの実体名に大文字を使わないでください

```suggestion
DROP TABLE product_category;
```

Reply:
これはなぜか補足すると、"" で囲まないとき大文字小文字を区別しない DB が多く存在するからです。つまり、アンスコなどで区切っておかないと `productcategory` でも `produCtcategory` でも良くなってしまうわけです。ならば、すべて小文字で統一して区切り文字を使ったほうがよいです (万が一そんなクエリがあったらレビューで弾きますが)

https://hito4-t.hatenablog.com/entry/2015/08/25/184107